app/scripts/metamask-controller.js

```diff
@@ -248,7 +249,7 @@ export default class MetamaskController extends EventEmitter {
     this.networkController.setInfuraProjectId(opts.infuraProjectId);
 
     // now we can initialize the RPC provider, which other controllers require
-    this.initializeProvider();
+    this.networkController.initializeProvider();
```

Comment:
We still need to call this to kick off the network controller initialization, but the `providerOptions` passed in before were _solely_ used to setup the metamask middleware. Now it doesn't need parameters anymore.